docs/memory.md

```diff
@@ -0,0 +1,104 @@
+---
+title: Using the Memory View
+---
+
+* toc
+{:toc}
+
+## What is it?
+
+Dynamically allocated Dart objects e.g., ```new ClassName()``` live in a portion of memory called the heap.
+
+DevTool’s Memory profile tab lets you peek at how an isolate is using memory at a given moment. This page also provides accumulator counts that you can use to study the rate of memory allocation. The accumulators are useful if you suspect that your application is leaking memory, or has other bugs relating to memory allocation.
+
+The memory view displays information about a Flutter application's memory usage as well the ability to detect memory leaks
+controlling the VMs allocation accumulator. Memory profiler consists of four parts, each increasing in granularity:
```

Comment:
Your right.  Is this better?

DevTools Memory tab lets you peek at how an isolate is using memory at a given moment. This pane, using Snapshot and Reset, can display accumulator counts.  The accumulators can be used to study the rate of memory allocations, if you suspect your application is leaking memory or has other bugs relating to memory allocation.

Memory profiling consists of four parts, each increasing in granularity:
- Memory usage overview chart
- Event timeline
- Classes allocated in the heap total instances, bytes allocated, accumulator of allocations since last reset
- Instances of a class type

Reply:
Memory usage overview chart is 'Memory overview chart'

Reply:
that sounds better! 👍